test/compiler.test.js

```diff
@@ -1564,6 +1564,21 @@ describe('compiler', function() {
         .to.throw(/path-does-not-exist/);
     });
 
+    it(
+      'does not fail when an optional middleware cannot be resolved',
+      function() {
```

Comment:
> jscs was complaining about the 80 characters things.

I see. Is the line still longer than 80 characters when you keep `it` + test name on the same line? I think it should fit 80 chars as long as `function() {` is on the next line.

>  Is the 80 characters limit an intentional choice?

Yes. First of all, the diff viewport on GitHub does not handle long lines well. One has to scroll horizontally, which is PITA. However, I think the viewport width is around 100 characters, so that does not justify the value 80.

I took 80 chars from Node's coding style. The limit is mostly historical, where it the goal was to support text terminals 80 chars x 25 lines, but in practice it works well on modern computers too. For example it allows me to edit two files side-by-side.